homeassistant/components/media_player/russound_rnet.py

```diff
@@ -91,33 +93,54 @@ def name(self):
 
     @property
     def state(self):
-        """Return the state of the device."""
-        return self._state
+        """Return the state of the device. directly from the device."""
+        if self._russ.get_power('1', self._zone_id) == 0:
```

Comment:
As pvizeli mentioned, no IO can be performed in the entity properties. Instead you need to create an update method, and store the state in `self._state`, to return here. Same for all other properties.

[There's an example here that should help](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/nad.py#L112), or you can look at other platforms for more examples.